File: GOL-Java/Wilk.java
package pl.edu.pg.eti.ksg.po.project2.zwierzeta;

import pl.edu.pg.eti.ksg.po.project2.Zwierze;
import pl.edu.pg.eti.ksg.po.project2.Swiat;
import pl.edu.pg.eti.ksg.po.project2.Punkt;

import java.awt.*;

public class Wilk extends Zwierze {
    private static final int zasieg = 1;
    private static final int szansa_ruchu = 1;
    private static final int sila = 9;
    private static final int inicjatywa = 5;

    public Wilk(Swiat swiat, Punkt pozycja, int turaUrodzenia) {
        super(TypOrganizmu.WILK, swiat, pozycja, turaUrodzenia, sila, inicjatywa);
        this.setZasiegRuchu(zasieg);
        this.setSzansaWykonywaniaRuchu(szansa_ruchu);
        setKolor(new Color(42, 38, 38));
    }

    @Override
    public String GetGatunek() {
        return "Wilk";
    }
}

File: GOL-Python/symulacja/organizmy/zwierzeta/wilk.py
from pomocnicze.wektor2d import Wektor2d
from symulacja.organizmy.zwierze import Zwierze


class Wilk(Zwierze):

    SILA = 9
    INICJATYWA = 5

    def __init__(self, polozenie: Wektor2d):
        super().__init__(polozenie, Wilk.SILA, Wilk.INICJATYWA)

    def rysowanie(self) -> str:
        return "#666564"

    def __str__(self):
        return "WILK"